docs/local-gadget.md

```diff
@@ -246,6 +246,24 @@ CONTAINER        T  PID     COMM             IP  SADDR                  DADDR
 test-container   C  11039   wget             4   172.17.0.2             188.114.96.7           57560   443
 ```
 
+### Trace/TcpConnect
+
+The tcpconnect trace gadget traces `connect()` system calls.
+
+```bash
+$ docker run -it --rm --name test-container busybox /bin/sh -c "wget http://www.example.com"
+Connecting to www.example.com (93.184.216.34:80)
+saving to 'index.html'
+index.html           100% |************************************************************************************************|  1256  0:00:00 ETA
+'index.html' saved
+```
+
+```bash
+$ sudo local-gadget trace tcpconnect
```

Comment:
What do you think about using the `--containername` to ensure we just capture the connection we generated? Otherwise, the output could contain many other entries 🙂 . I would suggest adding it for the trace/tcp gadget as well.

```suggestion
$ sudo local-gadget trace tcpconnect --containername test-container
```

Reply:
Could you please squash last two commits?

Reply:
sure thing! just updated the branch. thanks!